app/model/release.js

```diff
@@ -0,0 +1,139 @@
+// Imports for packages
+import mongoose from 'mongoose';
+import Promise from 'bluebird';
+import semver from 'semver';
+
+// Local imports
+import app from '~/'
+import Jenkins from './jenkins';
+import IssueSchema from './issue';
+import { BuildSchema } from './build';
+
+// Mongoose schema for releases
+const ReleaseSchema = mongoose.Schema({
```

Comment:
yeah, thanks for changing this — the inconsistent pluralization had been bugging me